athletes/admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from .models import *
+
+# Register your models here.
+admin.site.register(Trainer)
```

Comment:
Please use this format for register
```suggestion
@admin.register(Trainer)
class AdminCity(admin.ModelAdmin):
    list_display = ('id', 'first_name', 'last_name')
```